app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt

```diff
@@ -28,7 +28,7 @@ class ChatItemViewHolder(view: ItemChatItemBinding) : CommonViewHolder<ChatItemV
             } else {
                 firstEmojiTextView.visible()
                 avatar.gone()
-                firstEmojiTextView.text = item.firstEmoji
+                firstEmojiTextView.text = "a"
```

Comment:
a?

Reply:
@TruszczynskiA it's just a placeholder for the chat item. Just wanted to remove round avatar

Reply:
Why you need a placeholder here?